README.md

```diff
@@ -22,6 +22,26 @@ The extension will download and install a required PostgreSQL Tools Service pack
 Each operating system has a .vsix file with the required service included. Pick the file for your OS, download and install to get started.
 We recommend you choose a full release and ignore any alpha or beta releases as these are our daily builds used in testing.
 
+## Building from Source
```

Comment:
This is already mentioned in the wiki. Do we need this here ?

Reply:
Ah, I never found the wiki :)

Maybe we can make this a very short section like

```
## Building from Source

See the [project wiki](https://github.com/microsoft/azuredatastudio-postgresql/wiki/How-to-Contribute)
for build instructions.
```